src/definition/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rocket.chat/apps-ts-definition",
-  "version": "1.28.0-alpha",
+  "version": "1.29.0-alpha.0",
```

Comment:
Please revert this change. I know it is added automatically but it does not need to be committed by this PR.